custom_components/tuya_local/devices/smart_camera_sc116-wz3a.yaml

```diff
@@ -0,0 +1,511 @@
+name: Camera
+products:
+  - id: b60fclvud3xtcufm
+    manufacturer: Tuya
```

Comment:
Don't list Tuya as a manufacturer. Tuya does not generally manufacture end devices, only the WiFi/Zigbee/Bluetooth/Matter module in it, and they may provide reference designs for manufacturers to copy, but the end manufacturer is another company.
If you don't know the manufacturer, leave it out and let it default to showing Tuya.